src/System.Linq.Parallel/tests/QueryOperators/AllTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace Test
+{
+    public class AllTests
+    {
+        public static IEnumerable<object[]> OnlyOneData(object[] counts)
```

Comment:
Generates data for tests where a differing item (ie, `false` for a collection of `true` items) occurs at the start, middle, and end of the collection..

I probably should comment some of these more (at all?!)